config-templates/ibm/ibm-system-storage-block-csi-driver/1.11.1/custom-parameters.json

```diff
@@ -3,6 +3,64 @@
     "description": "The namespace where you want to create the deployment.",
     "displayname": "Namespace",
     "name": "namespace",
+    "templateKeyName": [
+      "namespace"
+    ],
+    "default": "default",
+    "required": "true",
+    "category": "config"
+  },
+  {
+    "description": "The name of the secret to create.",
+    "displayname": "Secret Name",
+    "name": "secret-name",
+    "templateKeyName": [
+      "secret-name"
+    ],
+    "required": "true",
+    "regex": "^([a-z0-9]|[a-z0-9][a-z0-9.-]*[a-z0-9]+)$",
+    "min-length": 1,
+    "max-length": 64,
+    "category": "config"
+  },
+  {
+    "description": "The address of the management server.",
+    "displayname": "Secret Management Address",
+    "name": "secret-management_address",
+    "templateKeyName": [
+      "secret-management_address"
+    ],
+    "required": "true",
+    "category": "config"
+  },
+  {
+    "description": "The username to use to authenticate to the management server.",
+    "displayname": "Secret Username",
+    "name": "secret-username",
+    "templateKeyName": [
+      "secret-username"
+    ],
+    "required": "true",
+    "category": "config"
+  },
+  {
+    "description": "The password to use to authenticate to the management server.",
+    "displayname": "Secret Password",
+    "name": "secret-password",
+    "templateKeyName": [
+      "secret-password"
+    ],
+    "required": "true",
```

Comment:
add `"mutable": "true"` if you want to allow user to rotate the password after deployment.

Reply:
done